core/src/test/java/training/taylor/timetracker/core/TrackerAddTest.java

```
@@ -96,6 +97,7 @@ public void testAddTimeEntry() {
     @Test(expected = NullPointerException.class)
     public void testAddNullTimeEntry() {
         entries.add(null);
+        throw new NullPointerException("throws null pointer exception");
```

need to throw this exception as it is expected as mentioned in Test parameter